src/landing_game_object.py

```diff
@@ -10,17 +10,33 @@ class LandingGameObject(pygame.sprite.Sprite):
     def __init__(
         self,
         image: pygame.surface,
-        pos: Vec2d = Vec2d(),
+        pos_pixel: Vec2d = Vec2d(),
     ) -> None:
         super().__init__()
         self.rect: pygame.Rect = pygame.Surface.get_rect(image)
-        self.pos = pos
-        self.rect.center = self.pos
+        self.rect.center = Vec2d(pos_pixel)
```

Comment:
The change in overlay from using `__dict__`  to using `dir()` and `getattr(obj, attr_name) ` as a subject could be integrated in PR #21 .
Maybe we also could think about giving a warning and some fallback value if a requested overlay attribute is not avaliable instead of letting the program crash.

Reply:
Arr!